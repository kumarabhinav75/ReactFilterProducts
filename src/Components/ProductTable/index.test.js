import React from 'react';
import renderer from 'react-test-renderer';
import ProductTable from './index';
import { shallow } from 'enzyme';
const PRODUCTS = [
  {category: 'Sporting Goods', price: '$49.99', stocked: true, name: 'Football'},
  {category: 'Sporting Goods', price: '$9.99', stocked: true, name: 'Baseball'},
  {category: 'Sporting Goods', price: '$29.99', stocked: false, name: 'Basketball'},
  {category: 'Electronics', price: '$99.99', stocked: true, name: 'iPod Touch'},
  {category: 'Electronics', price: '$399.99', stocked: false, name: 'iPhone 5'},
  {category: 'Electronics', price: '$199.99', stocked: true, name: 'Nexus 7'}
]; 
describe('ProductTable', () => {
  it('Should match the snapshot', () => {
    const tree = renderer.create(<ProductTable products = {PRODUCTS}/>);
    expect(tree).toMatchSnapshot();
  })
})