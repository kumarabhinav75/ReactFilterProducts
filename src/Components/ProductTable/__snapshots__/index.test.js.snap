// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductTable Should match the snapshot 1`] = `
<table>
  <thead>
    <tr>
      <th>
        Name
      </th>
      <th>
        Price
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th
        colSpan="2"
      >
        Sporting Goods
      </th>
    </tr>
    <tr>
      <td>
        Football
      </td>
      <td>
        $49.99
      </td>
    </tr>
    <tr>
      <td>
        Baseball
      </td>
      <td>
        $9.99
      </td>
    </tr>
    <tr>
      <td>
        Basketball
      </td>
      <td>
        $29.99
      </td>
    </tr>
    <tr>
      <th
        colSpan="2"
      >
        Electronics
      </th>
    </tr>
    <tr>
      <td>
        iPod Touch
      </td>
      <td>
        $99.99
      </td>
    </tr>
    <tr>
      <td>
        iPhone 5
      </td>
      <td>
        $399.99
      </td>
    </tr>
    <tr>
      <td>
        Nexus 7
      </td>
      <td>
        $199.99
      </td>
    </tr>
  </tbody>
</table>
`;
